spec/unit/pdk/util/puppet_version_spec.rb

```diff
@@ -24,9 +24,22 @@
   shared_context 'is a package install' do
     before(:each) do
       allow(PDK::Util).to receive(:package_install?).and_return(true)
+      allow(PDK::Util::RubyVersion).to receive(:versions).and_return('2.1.9' => '2.1.0', '2.4.3' => '2.4.0')
```

Comment:
This does not match actual behaviour of this method

Reply:
It does now (that method was just a stub that was going to be implemented in the next PR).